src/main/java/net/earthcomputer/clientcommands/command/ConnectFourCommand.java

```diff
@@ -0,0 +1,364 @@
+            };
+        }
+
+        public static Component name(byte piece) {
+            return switch (piece) {
+                case RED -> Component.translatable("connectFourGame.pieceRed");
+                case YELLOW -> Component.translatable("connectFourGame.pieceYellow");
+                default -> throw new IllegalStateException("Unexpected value: " + piece);
+            };
+        }
+
+        public static void render(GuiGraphics graphics, int x, int y, byte piece, boolean transparent) {
+            if (piece == 0) {
+                return;
+            }
+            int xOffset = switch (piece) {
+                case RED -> 0;
+                case YELLOW -> 16;
+                default -> throw new IllegalStateException("Unexpected value: " + piece);
+            };
+            graphics.blit(
+                RenderType::guiTextured,
+                ConnectFourGameScreen.PIECES_TEXTURE,
+                x,
+                y,
+                xOffset,
+                0,
+                ConnectFourGameScreen.PIECE_WIDTH,
+                ConnectFourGameScreen.PIECE_HEIGHT,
+                ConnectFourGameScreen.TEXTURE_PIECE_WIDTH,
+                ConnectFourGameScreen.TEXTURE_PIECE_HEIGHT,
+                ConnectFourGameScreen.TEXTURE_PIECES_WIDTH,
+                ConnectFourGameScreen.TEXTURE_PIECES_HEIGHT,
+                ((transparent ? 0x7F : 0xFF) << 24) | 0xFFFFFF // if I wanted, it could be (-1 >>> (transparent ? 1 : 0))
+            );
+        }
+    }
+
+    public static class ConnectFourGameScreen extends Screen {
+        private final ConnectFourGame game;
+
+        private static final ResourceLocation BOARD_TEXTURE = ResourceLocation.fromNamespaceAndPath("clientcommands", "textures/connect_four/board.png");
+        private static final ResourceLocation PIECES_TEXTURE = ResourceLocation.fromNamespaceAndPath("clientcommands", "textures/connect_four/pieces.png");
+
+        private static final int SCALE = 4;
+
+        private static final int TEXTURE_PIECE_WIDTH = 16;
+        private static final int TEXTURE_PIECE_HEIGHT = 16;
+        private static final int TEXTURE_BOARD_BORDER_WIDTH = 1;
+        private static final int TEXTURE_BOARD_BORDER_HEIGHT = 1;
+        private static final int TEXTURE_SLOT_BORDER_WIDTH = 1;
+        private static final int TEXTURE_SLOT_BORDER_HEIGHT = 1;
+        private static final int TEXTURE_SLOT_WIDTH = TEXTURE_PIECE_WIDTH + 2 * TEXTURE_SLOT_BORDER_WIDTH;
+        private static final int TEXTURE_SLOT_HEIGHT = TEXTURE_PIECE_HEIGHT + 2 * TEXTURE_SLOT_BORDER_HEIGHT;
+        private static final int TEXTURE_BOARD_WIDTH = TEXTURE_SLOT_WIDTH * ConnectFourGame.WIDTH + TEXTURE_BOARD_BORDER_WIDTH * 2;
+        private static final int TEXTURE_BOARD_HEIGHT = TEXTURE_SLOT_HEIGHT * ConnectFourGame.HEIGHT + TEXTURE_BOARD_BORDER_HEIGHT * 2;
+        private static final int TEXTURE_PIECES_WIDTH = TEXTURE_PIECE_WIDTH + TEXTURE_PIECE_WIDTH; // red and yellow
```

Comment:
```suggestion
        private static final int TEXTURE_PIECES_WIDTH = 2 * TEXTURE_PIECE_WIDTH; // red and yellow
```
Better to use multiplication if it's clearer